src/constants/routes.js

```diff
@@ -3,7 +3,7 @@ export const JOB = '/jobs/:job_id';
 export const MY_JOBS = '/my/jobs';
 export const SUBMIT_JOB = '/my/jobs/new'
 export const PREVIEW_JOB = '/my/jobs/new/preview'
-export const EDIT_JOB = '/my/jobs/job_id/edit'
+export const EDIT_JOB = '/my/jobs/:job_id/edit'
 export const ADMIN_LIST_JOBS = '/admin/jobs'
 export const ADMIN_PREVIEW_DRAFT_JOB = '/admin/jobs/:job_id'
```

Comment:
Yeah so my above comment raltes to these. The URLs should read :

`/my/job/new/preview`
`/my/job/:job_id/edit`
`/job/:job_id`

Does that make sense?

Reply:
I think the standard is to use the plural.  See number 3. here https://www.freecodecamp.org/news/rest-api-best-practices-rest-endpoint-design-examples/ But can change if you think that makes more sense for this site

Reply:
Ah ok, no worries then. We can keep it as it is